java/core/src/main/java/sleeper/core/schema/Schema.java

```diff
@@ -73,6 +73,18 @@ public List<PrimitiveType> getRowKeyTypes() {
         return rowKeyFields.stream().map(Field::getType).map(t -> (PrimitiveType) t).collect(Collectors.toList());
     }
 
+    /**
+     * A convenience method for cases where we've set up only one row key field.
+     *
+     * @return The row key field
+     */
+    public Field getSingleRowKeyField() {
+        if (rowKeyFields.size() != 1) {
+            throw new IllegalStateException("Cannot get single row key field, have " + rowKeyFields.size());
```

Comment:
The message would be better as "Cannot get single row key field, have " + rowKeyFields.size() + " row key fields."

Reply:
Done. I've also moved the method to the test code.